account_fiscal_year/views/account_fiscal_year_views.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  ~ Copyright 2020 Simone Rubino - Agile Business Group
+  ~ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+  -->
+<odoo>
+    <record id="account_fiscal_year_form_view" model="ir.ui.view">
+        <field name="name">Form view for account.fiscal.year</field>
+        <field name="model">account.fiscal.year</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="name" placeholder="Fiscal Year 2020" />
+                        <field name="date_from" />
+                        <field name="date_to" />
+                        <field name="company_id" groups="base.group_multi_company" />
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="account_fiscal_year_search_view" model="ir.ui.view">
+        <field name="name">Search view for account.fiscal.year</field>
+        <field name="model">account.fiscal.year</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name" />
+            </search>
+        </field>
+    </record>
+
+    <record id="account_fiscal_year_tree_view" model="ir.ui.view">
+        <field name="name">Tree view for account.fiscal.year</field>
+        <field name="model">account.fiscal.year</field>
+        <field name="arch" type="xml">
+            <list>
+                <field name="name" />
+                <field name="date_from" />
+                <field name="date_to" />
+                <field name="company_id" groups="base.group_multi_company" />
+            </list>
+        </field>
+    </record>
+
+    <record id="actions_account_fiscal_year" model="ir.actions.act_window">
+        <field name="name">Fiscal Years</field>
+        <field name="res_model">account.fiscal.year</field>
+        <field name="view_mode">tree,form</field>
```

Comment:
I am getting an error in runboat when accessing fiscal year from the menu, I think it could be the "tree" in here.